packages/driver/src/multi-domain/communicator.ts

```diff
@@ -125,6 +125,29 @@ export class PrimaryDomainCommunicator extends EventEmitter {
 export class SpecBridgeDomainCommunicator extends EventEmitter {
   private windowReference
 
+  private handleSubjectAndErr = (event, data) => {
```

Comment:
If needed this could be generalized further in the future, but for now we just need to handle the error and subject keys.